src/java.base/share/classes/java/lang/Class.java

```diff
@@ -1832,12 +1869,29 @@ private String getCanonicalName0() {
         }
     }
 
+    /**
+     * {@return {@code true} if and only if the underlying class
+     * is an unnamed class}
+     *
+     * @apiNote
+     * An unnamed class is not an {@linkplain #isAnonymousClass anonymous class}.
+     *
+     * @since 21
+     */
+    @PreviewFeature(feature=PreviewFeature.Feature.UNNAMED_CLASSES,
```

Comment:
Is there a JLS section to reference?

Reply:
Changing in primary PR - https://github.com/openjdk/jdk/pull/13689